modules/auxiliary/scanner/ssl/ssl_version.rb

```diff
@@ -56,6 +60,15 @@ def initialize
         [ 'CVE', '2013-2566' ],
         # LOGJAM
         [ 'CVE', '2015-4000' ],
+        # NULL ciphers
+        [ 'CVE', '2022-3358' ],
+        [ 'CWE', '319'],
+        # certificate expired
+        [ 'CWE', '295' ],
```

Comment:
CWE-324 seems more appropriate here. The issue is this is still focusing too much on the client not validating things correctly vs the server being misconfigured. CVE-324 is related to the use of a key past its expiration date which seems to describe this better.